### PR TITLE
Added NotePersonExample

### PR DESCRIPTION
**Pull request by gkowalski**
- Attempt to overide the Example text on the text field of a Note with a "subclass"

### CODE REVIEW

**Comment by gkowalski** [comment]
This works , I can build a flask app from it , but the resulting example text does not override the example for the Note / text field

**Comment by gkowalski** [comment]
I followed this page  https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/ to make this change.

**Comment by tschaffter** [comment]
@gkowalski Your PR fails because of a bug I may have introduce yesterday evening when trying to fix another one. I'll look at this later today.

**Comment by gkowalski** [comment]
I found an example in the slack group for openapi-generator which is alot of code just to change an example : 

```
 components:
  schemas:
    Type:
      type: string
      enum:
        - TYPE_1
        - TYPE_2
        - TYPE_3
    BaseObject:
      type: object
      discriminator:
        propertyName: type
        mapping:
          TYPE_1: '#/components/schemas/Type1'
          TYPE_2: '#/components/schemas/Type2'
          TYPE_3: '#/components/schemas/Type3'
      properties:
        type:
          $ref: '#/components/schemas/Type'
        baseField1:
          type: string
      required:
        - type
    Type1:
      allOf:
        - $ref: '#/components/schemas/BaseObject'
        - type: object
          properties:
            type1Field:
              type: integer
          required:
            - type1Field
    Type2:
      allOf:
        - $ref: '#/components/schemas/BaseObject'
        - type: object
    Type3:
      allOf:
        - $ref: '#/components/schemas/BaseObject'
        - type: object
          properties:
            type3Field:
              type: integer
              format: int64
          required:
            - type3Field
```

File: openapi/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+### Pre-tool setup
+
+Installation of the openapi-generator is documented [here](https://openapi-generator.tech/docs/installation) .
+
+#### Commands for Generating Stubbed out Flask app for each component
+
+Generate the Date Annotator Node
+
+    npm run build openapi/date-annotator/openapi.yaml
+    openapi-generator generate -i dist.yaml -g python-flask -o ~/PycharmProjects/nlp-sandbox-date-annotator-example/
```
Comment (tschaffter) [inline openapi/README.md:12]: There are two problems with this documentation:

- Developer will not use a `dist.yaml` file. Instead they will download the `openapi.yaml` that we host on GitHub Pages.
- The folder `PycharmProjects` is unique to your setup

Comment (tschaffter) [reply openapi/README.md:12]: Same comment for `openapi-generator` commands below

File: openapi/data-node/paths/notes.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ get:
             properties:
               items:
                 type: array
-                $ref: ../../commons/components/schemas/Note.yaml
+                $ref: '../../commons/components/schemas/NotePersonExample.yaml'
```
Comment (tschaffter) [inline openapi/data-node/paths/notes.yaml:25]: I don't like that.. Instead we should create a `Note.yaml` in the `schemas` folder of the person name annotator.

File: openapi/README.md (hunk 1 above)
Comment (tschaffter) [reply openapi/README.md:12]: Do we really want to give instruction to generate an implementation from a spec in development (dist.yaml)?

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Pull request closed without merging**